src/crawlee/_request.py

```diff
@@ -256,18 +247,6 @@ class Request(BaseRequestData):
     """A unique identifier for the request. Note that this is not used for deduplication, and should not be confused
     with `unique_key`."""
 
-    json_: str | None = None
-    """Deprecated internal field, do not use it.
-
-    Should be removed as part of https://github.com/apify/crawlee-python/issues/94.
-    """
-
-    order_no: Decimal | None = None
-    """Deprecated internal field, do not use it.
-
-    Should be removed as part of https://github.com/apify/crawlee-python/issues/94.
-    """
```

Comment:
This is a breaking change as `Request` is part of public interface. Please add exclamation mark to the PR title. And also mention the fields removal in the upgrading the guide (it should not affect much people, as they were internals, but someone can touch them).